python/docs/Makefile

```diff
@@ -4,7 +4,6 @@
 # You can set these variables from the command line.
```

Comment:
+1 for this change too although I didn't check the details. This Makefile was from the old sphinx example.